custom/enikshay/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+
+class IssuerId(models.Model):
+    """
+    This model is used to ensure unique, incrementing issuer IDs for users,
+    and to look up a user given an issuer ID.
+    obj.pk represents the serial issuer ID, later representations will be added as fields
```

Comment:
I thought about it, but it seemed surprising to have the object primary key renamed.  I'm guessing this will grow aliases for various representations of the issuer_id.  No strong feelings about it, though.